main.py

```diff
@@ -214,6 +222,9 @@ def quantize_aq(
                 )
 
         for names in sequential:
+            if loaded_layer:
+                print("Skipping quantization: loaded a previously quantized layer")
+                break
```

Comment:
Why this is inside the for loop: so as not to add another nested IF around the entire thing. This makes black do extreme line breaks.